cernopendata/modules/records/utils.py

```diff
@@ -187,6 +187,27 @@ def record_metadata_view(pid, record, template=None):
         ],
         pid=pid,
         record=record,
+        title="COD: " + record.get("title"),
+    )
+
+
+def term_metadata_view(pid, record, template=None):
```

Comment:
I'm not too happy having to add these two extra functions. If you have any other suggestions, I'm happy to hear them